### ISSUE TITLE
Can the video play when in viewport and how to have multiple videos?

### ISSUE THREAD

**Issue by raganello1** [linnett/backgroundVideo#35]
Hi,

using your plugin and it works great, trying to achieve a couple of things:

1. Is there a way to play the video just when in viewport and pause it when out? Also resume it when completely out of viewport would be a plus

2. Can I have more video on the same page using your plugin?

Any suggestion?
Thanks a lot

**Comment by linnett** [comment]
Yeah I could update this to do so. I'll add it to the feature list. When I get some time I'll do it. Alternatively if you want it before, please fork, do it yourself and put in a PR 👍

**Comment by raganello1** [comment]
Hi!
Trying to figure out how to do it, If I achieve this I will post it for sure.

Thanks!

**Comment by e0159000** [comment]
video play when in viewport
Very necessary
